src/main/java/com/epam/reportportal/jobs/clean/CleanStorageJob.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.reportportal.jobs.clean;
+
+import com.epam.reportportal.jobs.BaseJob;
+import com.epam.reportportal.storage.DataStorageService;
+import com.epam.reportportal.storage.MinioDataStorageService;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Removing data from storage.
+ */
+@Component
+public class CleanStorageJob extends BaseJob {
+
+    private final DataStorageService storageService;
+    private static final int CHUNK_SIZE = 1000;
+
+    public CleanStorageJob(JdbcTemplate jdbcTemplate, DataStorageService storageService) {
+        super(jdbcTemplate);
+        this.storageService = storageService;
+    }
+
+    @Scheduled(cron = "${rp.environment.variable.clean.storage.cron}")
+    @SchedulerLock(name = "cleanStorage", lockAtMostFor = "24h")
+    public void execute() {
+        logStart();
+        AtomicInteger counter = new AtomicInteger(0);
+
+        String chunkDataQuery = "DELETE FROM attachment_deletion WHERE id in " +
```

Comment:
Is this job going to remove only 1k of attachments and then finish?

Reply:
Yes, we will configure appropriate parameter for chunk, and configure amount of times to execute per day.(Based on our performance tests).
We haven't aim to clear all files in single day if it hard for our system, we can delete by appropriate chunk at one time and clearing out storage step by step, But i hope we can do it in 1 or 2 iterations(will be based on performace)

Reply:
So, we can provide batch size as a env variable and not change it directly in code.

Reply:
It's possible, but i think, that we can set it to big enough value after tests, and we don't need this to be a real changeable option.